docs/ovnkube.1

```diff
@@ -37,6 +37,9 @@ Initialize a gateway in the minion. Only useful with \fB--init-node\fR.
 The interface in minions that will be the gateway interface.  If none
 specified, then the node's interface on which the default gateway is
 configured will be used as the gateway interface. Only useful with
+\fB\--gateway-name\fR string
+The name for the gateway. Should be unique in the cluster. By default,
```

Comment:
The usage seems to suggest that this is a way to override the default gateway name. It would help to indicate how to achieve two gateways.

Reply:
The --init-gateways is what creates a gateway.  If you do not provide a name, it would be the name provided with --init-node. But I see that --init-gateways says that it should only be called with --init-node. I will fix it.